### PR TITLE
feat: Update Tomcat version and MongoDB Driver - MEED-7544 - Meeds-io/meeds#2417

### PR DESCRIPTION
**Pull request by boubaker**
This change will Tomcat version and MongoDB Driver to the declared versions inherited from Spring

### CODE REVIEW

**Comment by boubaker** [comment]
@rdenarie @bourasom @halaya need this PR approval to unblock CI/CD

**Review by hbenali — APPROVED** [review]

**Pull request merged**
